AU2/plugins/util/WantedManager.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+
+from typing import Dict, Tuple, List
+
+from AU2.database.AssassinsDatabase import ASSASSINS_DATABASE
+from AU2.database.GenericStateDatabase import GENERIC_STATE_DATABASE
+from AU2.database.model import Event, Assassin
+
+
+class WantedManager:
```

Comment:
It would be nice to get some tests for this class (see the [testing framework](https://github.com/jsyiek/AU2/pull/30/files))

Reply:
E.g. https://github.com/jsyiek/AU2/blob/master/AU2/test/plugins/util/custom_plugins/test_CompetencyPlugin.py#L22